tdrs-backend/tdpservice/users/permissions.py

```diff
@@ -19,76 +79,101 @@ def is_own_stt(request, view):
     user_stt = request.user.stt_id if hasattr(request.user, 'stt_id') else None
 
     return bool(
-        is_data_analyst and
         user_stt is not None and
         (requested_stt in [None, str(user_stt)])
     )
 
 
-def is_in_group(user, group_name):
-    """Take a user and a group name, and returns `True` if the user is in that group."""
-    return user.groups.filter(name=group_name).exists()
+class DjangoModelCRUDPermissions(permissions.DjangoModelPermissions):
+    """The request is authorized using `django.contrib.auth` permissions.
 
+    See: https://docs.djangoproject.com/en/dev/topics/auth/#permissions
 
-class IsUser(permissions.BasePermission):
-    """Object-level permission to only allow owners of an object to edit it."""
+    It ensures that the user is authenticated, and has the appropriate
+    `view`/`add`/`change`/`delete` permissions on the model.
 
-    def has_object_permission(self, request, view, obj):
-        """Check if user has required permissions."""
-        return obj == request.user
+    This permission can only be applied against view classes that
+    provide a `.queryset` attribute.
+    """
 
+    def __init__(self):
+        # Use deepcopy to prevent overwriting the parent class perms_map
+        self.perms_map = deepcopy(self.perms_map)
+        # We also want to check for the `view` permission for GET requests.
+        self.perms_map['GET'] = ['%(app_label)s.view_%(model_name)s']
```

Comment:
The only difference between this and the base class is the checking of the `view` permission for GET requests. I'm not really sure why they didn't implement this by default, as they even left instructions in the code for how to do this :shrug: